Formula/conman.rb

```diff
@@ -0,0 +1,36 @@
+class Conman < Formula
+  desc "Serial console management program supporting a large number of devices"
+  homepage "https://github.com/dun/conman"
+  url "https://github.com/dun/conman/archive/conman-0.3.1.tar.gz"
+  sha256 "cd47d3d9a72579b470dd73d85cd3fec606fa5659c728ff3c1c57e970f4da72a2"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "freeipmi"
+
+  def install
+    system "./bootstrap"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  def caveats
+    <<~EOS
+      Before starting the conmand service, configure some consoles in #{etc}/conman.conf.
+    EOS
+  end
```

Comment:
If this is in the upstream, we probably dont need this in homebrew-core, if not, we can contribute back to the upstream? Might be good to use a general sample `conman.conf` if exists.

Reply:
There is a sample configuration, but it does not define any consoles.  You need to have a console to connect to and there really isn't a default.

This package is useful w/o configuring conmand, the conman client can be configured to talk to a remote server.

Reply:
👍